docs/src/main/sphinx/security/ldap.rst

```diff
@@ -243,15 +244,20 @@ Environment configuration
 TLS configuration
 ~~~~~~~~~~~~~~~~~
 
-Access to the Trino coordinator should be through HTTPS when using LDAP
-authentication. The Trino CLI can use either a :ref:`Java Keystore
-<server_java_keystore>` file or :ref:`Java Truststore <cli_java_truststore>`
-for its TLS configuration.
+When using LDAP authentication, access to the Trino coordinator must be
+through :doc:`HTTPS/TLS </security/https>`.
 
-If you are using a keystore file, it can be copied to the client machine and used
-for its TLS configuration. If you are using truststore, you can either use
+The Trino CLI can use a :ref:`PEM certificate <pem_certs>`, a :ref:`Java
+keystore <server_java_keystore>` file, or :ref:`Java truststore
+<cli_java_truststore>` for its TLS configuration. The most convenient option is
+to purchase a PEM certificate from a Certificate Authority already listed in the
+Java truststore.
+
+If you are using a keystore file, it can be copied to the client machine and
+used for its TLS configuration. If you are using truststore, you can either use
 default Java truststores or create a custom truststore on the CLI. We do not
-recommend using self-signed certificates in production.
+recommend using :ref:`self-signed certificates <self_signed_limits>` in
+production.
```

Comment:
Agree on the concept, but not the method. We're certainly not going to _copy_ the same info to several places, because that guarantees version skew over time as one copy is updated and others are forgotten. Sphinx isn't very good about importing snippets into doc pages, but that's the right way going forward. Can we postpone doing this for a future PR when we have a better way to import a section into different pages?